### ISSUE TITLE
leaderboard

### ISSUE THREAD

**Issue by phylyc** [arenarium/battleground_core#63]

**Comment by vincentropy** [comment]
Why is this blocking #97 ?

**Comment by phylyc** [comment]
Maybe I should have been more specific in #97, but the way I was thinking about the selection was based on some metric (e.g. elo rating), and this metric defines a leaderboard. Thus, we need a leaderboard (or rather the metric to it) before we can work on #97.

**Comment by vincentropy** [comment]
That makes sense. I think we can implement that pretty easily since this information is already in the database. We just need a query to extract it. I'll create a separate issue to create a python function that retrieves this ranking.

**Comment by vincentropy** [comment]
Moved issue to UI repo.
